sycl/test-e2e/Matrix/Inputs/joint_matrix_half_impl.hpp

```diff
@@ -116,33 +134,60 @@ int main() {
       continue;
 
     if (combinations[i].nsize == 0) { // Intel AMX
-      test<float, half, 2, /*TM*/ 16, /*TN*/ 16, /*TK*/ 32>();
+      test<half, float, float, 2, /*TM*/ 16, /*TN*/ 16, /*TK*/ 32>();
       break;
     }
 
     if (combinations[i].nsize == 16) { // architecture::intel_gpu_pvc
-      test<float, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
-      // test<half, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<half, float, float, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
+      test<half, half, half, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
 
       // This combination is not currently supported for sub group size = 32 in
       // IGC
 #if (!defined(SG_SZ) || SG_SZ != 32)
-      // test<float, half, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
-      // test<half, half, /*TM*/ 16, /*TN*/ 16, /*TK*/ 16>();
-      // test<float, half, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
-      // test<half, half, /*TM*/ 1, /*TN*/ 64, /*TK*/ 16>();
-      // test<float, half, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
-      // test<half, half, /*TM*/ 32, /*TN*/ 64, /*TK*/ 16>();
-      // test<float, half, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
-      // test<half, half, /*TM*/ 1, /*TN*/ 64, /*TK*/ 32>();
-      // test<float, half, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
-      // test<half, half, /*TM*/ 32, /*TN*/ 64, /*TK*/ 32>();
+#if (defined(ACC_BFLOAT16))
+      // 8x16x16
+      test<half, half,  float, 2, /*TM*/ 8, /*TN*/ 16, /*TK*/ 16>();
```

Comment:
The tests under this macro are not being currently exposed in any `.cpp` file and hence not running. is it intentional?